<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fullWindowInfoClass</class>
 <widget class="QWidget" name="fullWindowInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>140</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>fullWindowInfo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000;</string>
  </property>
  <widget class="QPushButton" name="pushButton_main">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>141</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0px 0px 40px 0px;
color: #ffffff;
font: 38pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>   Finish</string>
   </property>
   <property name="icon">
    <iconset resource="FTRT.qrc">
     <normaloff>:/Res/finish.svg</normaloff>:/Res/finish.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>90</y>
     <width>491</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>%1 files successfully processed.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FTRT.qrc"/>
 </resources>
 <connections/>
</ui>
